plugins/dictionary/plugin.go

```diff
@@ -136,14 +136,9 @@ func (p *plugin) loadFile() error {
 				continue
 			}
 
-			idx := indexOf(p.definitions, line)
-			if idx > -1 {
-				indices = append(indices, idx)
-				continue
-			}
-
-			indices = append(indices, len(p.definitions))
 			p.definitions = append(p.definitions, line)
+			idx := indexOf(p.definitions, line)
+			indices = append(indices, idx)
```

Comment:
Provided that the new definition is not a duplicate, already present in the `p.definitions` list, `idx` will always be equal to `len(p.definitions) - 1` here. So the `idx := indexOf(...` line can be removed and `indices = append(indices, idx)` changed to: `indices = append(indices, len(p.definitions) - 1)`.